src/WebJobs.Script.WebHost/WebScriptHostManager.cs

```diff
@@ -394,9 +420,39 @@ protected override void OnHostStarted()
                 _secretManager.PurgeOldSecretsAsync(Instance.ScriptConfig.RootScriptPath, Instance.TraceWriter, Instance.Logger);
             }
 
+            if (Microsoft.Azure.WebJobs.Script.WebHost.Management.FunctionsSyncManager.IsSyncTriggersEnvironment(_settingsManager))
+            {
+                // create a onetime invocation timer
```

Comment:
I never reviewed the v2 change so pardon the question -- but we only run this once, 30 seconds after startup? For some reason I thought it was running periodically checking for any missed changes.

Reply:
Just run on host startup, which is a natural periodicity